tool/sync_default_gems.rb

```diff
@@ -600,7 +627,7 @@ def sync_default_gems_with_commits(gem, ranges, edit: nil)
         puts "Remove files added to toplevel: #{toplevels.join(', ')}"
         system(*%w"git rm -r --", *toplevels)
       end
-      tools = changed.select {|f|f.start_with?("test/fixtures/", "test/lib/", "tool/")}
+      tools = changed.select {|f|f.start_with?("test/lib/", "tool/")}
```

Comment:
Ditto.

Reply:
I fixed the other one, but it seems like you commented on an outdated diff for this one.